jwst/outlier_detection/outlier_detection_scaled_step.py

```diff
@@ -77,8 +77,7 @@ def process(self, input):
                         **pars)
             step.do_detection()
 
-            for model in self.input_models:
-                model.meta.cal_step.outlier_detection = 'COMPLETE'
+            self.input_models.meta.cal_step.outlier_detection = 'COMPLETE'
```

Comment:
This change was required since it turned out that the input to this specific step (as implemented now) is always a single CubeModel.  Therefore, the assignment needed to be revised to reflect that.